monkey/monkey_island/cc/resources/agent_logs.py

```diff
@@ -0,0 +1,26 @@
+from http import HTTPStatus
+
+from flask import request
+
+from monkey_island.cc.repository import IAgentLogRepository
+from monkey_island.cc.resources.AbstractResource import AbstractResource
+from monkey_island.cc.resources.request_authentication import jwt_required
+
+
+class AgentLogs(AbstractResource):
+    urls = ["/api/agent-logs/<string:agent_id>"]
+
+    def __init__(self, agent_log_repository: IAgentLogRepository):
+        self._agent_log_repository = agent_log_repository
+
+    @jwt_required
+    def get(self, agent_id: str):
+        log_contents = self._agent_log_repository.get_agent_log(agent_id)
+
+        return log_contents, HTTPStatus.OK
+
+    def put(self, agent_id: str):
+        log_contents = request.json["log_contents"]
```

Comment:
Does the request body need to look like `{"log_contents": "LOG CONTENTS"}`? Can the request body just be `"log_contents"`? A single string is valid JSON.